src/components/FlameGraph/FlameGraph.jsx

```diff
@@ -171,19 +180,39 @@ class FlameGraph extends Component {
         this.props.history.push({search: params.toString(),});
     }
 
+    handleLayoutChange(event, data) {
+        this.setState({layout: data.value}, () => {
+            this.state.chart
+                .inverted(this.state.layout == 'icicle')
+                .resetZoom()
+            localStorage.setItem('layout', this.state.layout)
+        })
+    }
+
     render() {
         const searchButton = 
         <Button color='red' size='small' onClick={this.handleSearchClick}>
             <Button.Content>Search</Button.Content>
         </Button>
-        
+        const layoutOptions = [
```

Comment:
The `Layout` word is redundant. I would suggest removing it, keeping Flame and Icicle only, to reduce the size of the dropdown.